src/pages/sc-cycle-counts/Forms/Sites.js

```diff
@@ -0,0 +1,208 @@
+import { useContext, useEffect } from 'react'
+import { useForm } from 'src/hooks/form'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { SCRepository } from 'src/repositories/SCRepository'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+
+const Sites = ({ store, maxAccess, labels, setRefreshController, refreshController }) => {
+  const { recordId, isPosted, isClosed } = store
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { formik } = useForm({
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues: {
+      search: '',
+      rows: [
+        {
+          id: 1,
+          siteId: '',
```

Comment:
Is it necessary to add to formik initial values
 siteName: '',
 siteRef: '',
 statusName: '',